DirectEmployers/MyJobs
# [PD 3093] export failing on production

## What's this PR do?
PR fixes the exports failing on production for Lisa Harr

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics application signed in as Lisa Harr.

## How should this be manually tested?
Pull Down the repository. You will need access to login as the customer Lisa Harr from BELOW. When you do that, if you go into Analytics in production and click to export the data as csv or excel, it should export now instead of do nothing.

## Any background context you want to provide?
Lisa Harr at VSP Global reported that exports do not work in analytics. They work for me in production, but once I impersonated her account, I was able to duplicate the error. There are no issues reported in the console, but clicking on export does nothing.
